### PR TITLE
Added Focal-specific files

### PR DESCRIPTION
**Pull request by ivaradi**
These files are needed to properly build for Focal.

### CODE REVIEW

File: admin/linux/debian/debian.focal/control [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Package: nextcloud-client-l10n
+Architecture: all
+Depends: ${misc:Depends}
+Description: Nextcloud client internatialization files
+ The translation files.
+
+Package: nextcloud-client-nautilus
+Architecture: all
+Depends: nextcloud-client (>=${binary:Version}), libnextcloudsync0, python-nautilus, nautilus, ${misc:Depends}
```
Comment (tintou) [inline admin/linux/debian/debian.focal/control]:
```suggestion
Depends: nextcloud-client (>=${binary:Version}), libnextcloudsync0, python3-nautilus, nautilus, ${misc:Depends}
```

File: admin/linux/debian/debian.focal/control [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Package: nextcloud-client-nemo
+Architecture: all
+Depends: nextcloud-client (>=${binary:Version}), libnextcloudsync0, python-nemo | nemo-python, nemo, ${misc:Depends}
+Description: Nemo plugin for Nextcloud
+ This package contains a Nemo plugin to display
+ synchronization status icons for Nextcloud files.
+
+Package: nextcloud-client-caja
+Architecture: all
+Depends: nextcloud-client (>=${binary:Version}), libnextcloudsync0, python-caja, caja, ${misc:Depends}
```
Comment (tintou) [inline admin/linux/debian/debian.focal/control]:
```suggestion
Depends: nextcloud-client (>=${binary:Version}), libnextcloudsync0, python3-caja, caja, ${misc:Depends}
```

File: admin/linux/debian/debian.focal/control [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+               libhttp-dav-perl,
+               libinotify-dev [kfreebsd-any],
+               libqt5svg5-dev,
+               libqt5webkit5-dev,
+               libsqlite3-dev,
+               libssl-dev (>= 1.1.0),
+               zlib1g-dev,
+               optipng,
+               pkg-kde-tools,
+               python-sphinx | python3-sphinx,
```
Comment (tintou) [inline admin/linux/debian/debian.focal/control]:
```suggestion
               python3-sphinx,
```

File: admin/linux/debian/debian.focal/control (hunk 1 above)
Comment (ivaradi) [reply admin/linux/debian/debian.focal/control]: Fixed

File: admin/linux/debian/debian.focal/control (hunk 2 above)
Comment (ivaradi) [reply admin/linux/debian/debian.focal/control]: Fixed

File: admin/linux/debian/debian.focal/control (hunk 3 above)
Comment (ivaradi) [reply admin/linux/debian/debian.focal/control]: Fixed

**Comment by tintou** [comment]
Looks good to me

**Pull request merged**